README.md

```diff
@@ -100,6 +100,7 @@ Third-party A-Frame components (of the entity-component-system pattern).
 - [Fence](https://github.com/atomicguy/aframe-fence-component) - Set boundaries on entity position
 - [Mouse Cursor](https://github.com/mayognaise/aframe-mouse-cursor-component) - Use mouse as a direct pointer
 - [Drag Look](https://github.com/mayognaise/aframe-drag-look-controls-component) - Look controls in the opposite direction with changing cursor style
+- [Ocean Primitive](https://github.com/donmccurdy/aframe-extras/blob/master/src/primitives/a-ocean.js) - Flat-shaded ocean primitive, with animated waves. Based on a [Codrops THREE.js tutorial](http://tympanus.net/codrops/2016/04/26/the-aviator-animating-basic-3d-scene-threejs/).
```

Comment:
Maybe just list it as `Ocean` and mention there's a primitive too.

No period for consistency